src/redux/books/books.js

```diff
@@ -0,0 +1,40 @@
+import { createSlice } from '@reduxjs/toolkit';
+import { v4 as uuid4 } from 'uuid';
+
```

Comment:
Great job @Linushaddai99 🤝

- [ ] [Optional] Kindly define ADD_BOOK and REMOVE_BOOK as action types so it will be something like: `const ADD_BOOK = bookstore/books/ADD_BOOK` and `const REMOVE_BOOK = bookstore/books/REMOVE_BOOK`.